hub/jupyterhub_config.py

```diff
@@ -13,6 +13,28 @@
 # Our simplest user image! Optimized to just... start, and be small!
 c.KubeSpawner.singleuser_image_spec = 'yuvipanda/simple-singleuser:v1'
 
+# Configure dynamically provisioning pvc
+c.KubeSpawner.pvc_name_template = 'claim-{username}-{userid}'
```

Comment:
Give a unique name per user to a persistent volume claim (PVC) so that kubespawner can either start a PVC (if it doesn't exist) or mount to the existing one with that name ```'claim-{username}-{userid}'```.

This allows us to, per user basis, to attach to the correct PVC. A PVC is launched in Kubespawner with the following code:

```if self.storage_class:
            pvc_manifest = self.get_pvc_manifest() // json blob specifying a pvc. See objects.py
            try: // use api to try to create a pvc with the specs defined in pvc_manifest
                yield self.httpclient.fetch(self.request(
                    url=k8s_url(self.namespace, 'persistentvolumeclaims'),
                    body=json.dumps(pvc_manifest),
                    method='POST',
                    headers={'Content-Type': 'application/json'}
                ))
            except:
                self.log.info("Pvc " + self.pvc_name + " already exists, so did not create new pod.")
```